### PR TITLE
Moved JupySQL initialisation into a separate notebook

### LINKED ISSUE #108: Move the JupySQL activation to a separate notebook

### PR DESCRIPTION
**Pull request by ahsimb**
Closes #108

### CODE REVIEW

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb [hunk 1]
```diff
@@ -33,19 +33,6 @@
     "display(get_access_store_ui('../'))"
```
Comment (tkilias) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:33]: <p>The open_pyexasol_connection should already activate the language container.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/jupysql_init.ipynb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{
```
Comment (tkilias) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/jupysql_init.ipynb:1]: <pre class="ql-syntax" spellcheck="false">model loading functions
</pre><p>I think, this a copy paste typo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb (hunk 1 above)
Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:33]: <p>How come???</p>

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/jupysql_init.ipynb (hunk 2 above)
Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/jupysql_init.ipynb:1]: <p>Yep, well spotted! Thanks</p>

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb (hunk 1 above)
Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:33]: The helper notebook opening the JupySQL includes the container activation. The text generation is the only notebook in the transformers series where I don't use the JupySQL. Here it's a normal pyexasol connection.

Comment (tkilias) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:33]: it was my assumption, from what we have discussed last year, that open_pyexasol_connection does the activation, but maybe that got lost

Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:33]: No, open_pyexasol_connection_connection does just what the name says - opens the connection. I categorically object to loading it with other stuff.

Comment (tkilias) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/transformers/text_generation.ipynb:33]: ok, I can live with that for the moment

**Pull request merged**